source/common/http/conn_manager_impl.cc

```diff
@@ -772,6 +772,14 @@ void ConnectionManagerImpl::ActiveStream::decodeHeaders(RequestHeaderMapPtr&& he
   ScopeTrackerScopeState scope(this,
                                connection_manager_.read_callbacks_->connection().dispatcher());
   request_headers_ = std::move(headers);
+  auto host_ = connection_manager_.read_callbacks_->upstreamHost();
+
+  if (host_ != nullptr) {
+    if (host_->cluster().requestResponseSizeStats().has_value()) {
+      host_->cluster().requestResponseSizeStats()->get().upstream_rq_headers_size_.recordValue(
```

Comment:
as `requestResponseSizeStats()` may return nullopt, this may be source of npe

Reply:
nvm, missed if case above

Reply:
> source of npe

Fixed the npe earlier. All checks are passing now :rocket: